ppl/src/main/antlr/OpenSearchPPLParser.g4

```diff
@@ -521,32 +521,41 @@ dateTimeFunctionName
     | DATE
     | DATE_ADD
     | DATE_FORMAT
+    | TIME_FORMAT
```

Comment:
Please keep it in alphabetical order

Reply:
fixed in https://github.com/Bit-Quill/opensearch-project-sql/pull/281/commits/7b927df652fd09ef0e74315eb6a2abfbffe1f8f3